traveler-ios-ui/TravelerKitUI/ViewControllers/PartnerOfferingQuantityViewController.swift

```diff
@@ -55,20 +56,28 @@ open class PartnerOfferingQuantityViewController: UIViewController {
     }
 
     @IBAction func stepperValueDidChange(_ sender: Stepper) {
+        totalOptions.removeAll()
```

Comment:
If you make: `totalOptions: [[PartnerOffering]]`
you can simplify this function to:
       ```
 if totalOptions.count < sender.value {
            totalOptions.append(selectedOptions!)
        } else {
            totalOptions = totalOptions.dropLast()
        }        
```

And just flatmap `totalOptions` before sending the request

Reply:
Decided to change the place where im doing the appending. Realized its unnecessary to do it every time the stepper updates. Only matters when submitting. So before submitting Im iterating over the options and multiplying them to the stepper value. That way this is only done once.